orchestration/src/main/java/com/sap/ai/sdk/orchestration/OrchestrationClientException.java

```diff
@@ -22,12 +24,29 @@ public class OrchestrationClientException extends ClientException {
    * Retrieves the {@link ErrorResponse} from the orchestration service, if available.
    *
    * @return The {@link ErrorResponse} object, or {@code null} if not available.
+   * @since 1.10.0
    */
   @Beta
   @Nullable
   public ErrorResponse getErrorResponse() {
     final var clientError = super.getClientError();
-    if (clientError instanceof OrchestrationError orchestrationError) {
+    if (clientError instanceof OrchestrationError.Synchronous orchestrationError) {
+      return orchestrationError.getErrorResponse();
+    }
+    return null;
+  }
+
+  /**
+   * Retrieves the {@link ErrorResponseStreaming} from the orchestration service, if available.
+   *
+   * @return The {@link ErrorResponseStreaming} object, or {@code null} if not available.
+   * @since 1.10.0
+   */
+  @Beta
+  @Nullable
+  public ErrorResponseStreaming getErrorResponseStreaming() {
```

Comment:
(Comment/Todo)

I want to check whether / how to this impacts API usability.
I.e. the user will have to choose the correct method at design time - Is it obvious?

Reply:
(Haven't tried anything yet but...)
If the user runs `streamChatCompletion()`, it would not be obvious why `getErrorResponse()` wouldn't work naturally.
Often, we cannot assume users are studying all available methods before choosing one.